site/content/blog/2021-06-01-whats-new-in-svelte-june-2021.md

```diff
@@ -0,0 +1,55 @@
+---
+title: What's new in Svelte: June 2021
+description: Progress on SvelteKit V1 and tighter TypeScript/Svelte integrations in language tools
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+This month, we saw lots of contributions to SvelteKit and its docs. The language tools also got some new features alongside some hydration bug fixes in Svelte core. Let's jump into the updates...
+
+## New in SvelteKit
+- `svelte.config.js` config files can now be loaded in ESM format (`.js` instead of `.cjs` - either can be used). It is recommended to use the `.js` format going forward.
```

Comment:
> either can be used

I don't believe either can be used. I think it's forced to be `.js` and you'll get a warning or error when starting if it's not